R/readNavigate.R

```diff
@@ -0,0 +1,47 @@
+#' readNavigate
+#'
+#' Read in navigate from iiasa.
```

Comment:
change to "Read in NAVIGATE project model scenarios from IIASA."

Reply:
ok